css/style.css

```diff
@@ -332,6 +334,391 @@ body {
   background: #2c2c2c;
 }
 
+/* Второй блок */
+
+.video {
+  min-height: 494px;
+  min-width: 1030px;
+  background: #000;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  margin-bottom: 77px;
+}
+
+.video__frame {
+  background: url(../img/video-bg.png) no-repeat;
+  height: 361px;
+  width: 644px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-left: 12px;
+}
+
+.video__btn {
+  height: 47px;
+  width: 47px;
+  background: #fff;
+  border-radius: 50%;
+  box-shadow: 0 0 0px 4px #ffffff78;
+  border: none;
+  position: relative;
+  outline: none;
+}
+
+.video__btn::before {
+  display: block;
+  content: '';
+  position: absolute;
+  left: 16px;
+  bottom: 15px;
+  background: url('../icons/Rectangle.png');
+  width: 16px;
+  height: 16px;
+}
+
+.video__btn:hover {
+  cursor: pointer;
+  box-shadow: inset 0 0 0px 2px #5452524a;
+}
+
+.video__btn:active {
+  cursor: pointer;
+  box-shadow: inset 0 0 0px 2px #ffffff00;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#ffffff00" to be "#fff0" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5612618)